core/include/traccc/clusterization/impl/measurement_creation.ipp

```diff
@@ -93,6 +93,13 @@ TRACCC_HOST_DEVICE inline void fill_measurement(
 
         // For the ambiguity resolution algorithm, give a unique measurement ID
         m.measurement_id = measurement_index;
+
+        // Adjust the measurement object for 1D surfaces.
+        if (module.pixel.dimension == 1) {
+            m.meas_dim = 1;
+            m.local[1] = 0.f;
+            m.variance[1] = 1000.f;
```

Comment:
Note that I set this variance sort of arbitrarily here. :thinking:

For the ODD the correct values, for the different modules, would be: `2024.39306640625`, `972.7376098632812` and `2029.76708984375`.

If you think that would be useful to have, I could add it as yet one more parameter to `traccc::cell_module`. :thinking:

Reply:
Decided to make it "more correct" in the end.

With SoA containers hopefully the overhead of all of this will be minimal in the end. :thinking: